### ISSUE TITLE
Does  LogitMarginL1 fit for BCE Loss?

### ISSUE THREAD

**Issue by myt889** [by-liu/MbLS#3]
Hi, thank you for your great work. I wonder know if your  LogitMarginL1 is fit for multi-label classification task? Thank you so much.

**Comment by by-liu** [comment]
Hello. That's a very good question.

From my rough view, it is available if the strategry is one-vs-rest.
Under one-vs-rest, we train binary classification for each class or node, and apply sigmoid function on each logit node which is a special case of softmax. Then the margin constraint could be included on each node corresponding to each classifier.

However, for sure, there are some additional work to update the  code to cover this scenario. I think it should not be very hard. 

If you are interested to try it, let me know if you need my support.

Best
